pkg/multitenant/mtinfopb/info.proto

```diff
@@ -58,6 +63,9 @@ message TenantInfo {
   optional cockroach.multitenant.tenantcapabilitiespb.TenantCapabilities capabilities = 6 [
     (gogoproto.nullable) = false
   ];
+
+  // available tag: 3.
+  // available tag: 7 or above.
```

Comment:
There is some precedent for "Next ID: x" with various capitalization as the last line of a message elsewhere in the codebase.